docs/commands/push.md

```diff
@@ -84,6 +84,7 @@ destination      | string    | Required. The location in the API registry where
 --branch, -b    | string  | Optional. The branch where your API definition will be pushed or upserted. Default value is `main`.  |
 --help       | boolean | Optional. Help output for the command.  |
 --run-id       | string  | Optional. Specify the ID of the CI job that the current push will be associated with. See [the Run ID section](#run-id) for more information.  |
+--skip-decorator | [string] | Optional. Ignore certain decorators. See the [Skip preprocessor, rule, or decorator section](#skip-preprocessor-rule-or-decorator).
```

Comment:
```suggestion
--skip-decorator | [string] | Optional. Ignore one or more decorators. See the [Skip preprocessor, rule, or decorator section](#skip-preprocessor-rule-or-decorator) for usage examples.
```

Reply:
Added this suggested change with commit.